### PR TITLE
Added basic cluster-components structure

### PR DESCRIPTION
**Pull request by Cellebyte**

### CODE REVIEW

File: cluster-components/environments/prd/helmrepositories/kyverno_schiff-system.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: kyverno
+  namespace: schiff-system
+spec:
+  interval: 5m
+  url: https://harbor.sce-dcn.net/chartrepo/helm-kyverno
```
Comment (MaxRink) [inline cluster-components/environments/prd/helmrepositories/kyverno_schiff-system.yaml:8]: sce-dcn.net needs replacement, too

**Comment by Cellebyte** [comment]
@MaxRink let us squash and merge.

**Comment by vukg** [comment]
That was lots of yaml :)

**Comment by Cellebyte** [comment]
@vukg they asked for examples they get one :)

**Comment by vukg** [comment]
> @vukg they asked for examples they get one :)

What does a Yaml engineer do? Yaml mining 😇

File: cluster-components/customers/customer-shortname/prd/das-schiff-rbac-manager-tenant.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    roleBindings:
+      - clusterRole: poweruser-plus
+        namespaceSelector:
+          matchExpressions:
+            - key: platform.das-schiff.io/owner
+              operator: NotIn
+              values:
+                - das-schiff
+    clusterRoleBindings:
+      - clusterRole: customer-m2m # temporary non production for the A4 guys. As in the email with Robert Souckup we will not operate it or support it.
```
Comment (schrej) [inline cluster-components/customers/customer-shortname/prd/das-schiff-rbac-manager-tenant.yaml:19]:
```suggestion
      - clusterRole: customer-m2m
```

**Comment by schrej** [comment]
#3

**Pull request merged**